tools/template

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo '# Add sumologic repository'
+helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection 1>/dev/null 2>&1
+echo '# Update repository'
+helm repo update 1>/dev/null 2>&1
+echo '# Generating template'
+helm template sumologic/sumologic "$@"
```

Comment:
When we run `helm template` on the sumologic repo, does the generated template also contain the dependency chart contents? or will the user still need to run `helm template` for prometheus, fluent-bit, etc separately?

Reply:
It contains, but prometheus operator creates crd by itself, what leads to errors during first apply:
```
unable to recognize no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"
unable to recognize "/sumologic/sumologic.yaml": no matches for kind "Prometheus" in version "monitoring.coreos.com/v1"
unable to recognize "/sumologic/sumologic.yaml": no matches for kind "PrometheusRule" in version "monitoring.coreos.com/v1"
```
but after prometheus operator create the crds(~20s) next apply goes smooth

Reply:
Any chance to ensure the CRDs get applied first to prevent errors?

Reply:
As it's created by the operator 's pod we cannot. We could take it from running cluster and put into scripts, but I don't feel its safe way and leads to complications

Reply:
Also I think that is the reason why `helm delete` does not remove prometheus crds

Reply:
Here is potential solution:
https://github.com/helm/charts/tree/master/stable/prometheus-operator#helm-fails-to-create-crds

We can put the crds definitions on the top of the sumologic.yaml file